meson.build

```diff
@@ -36,6 +36,7 @@ libendless = dependency('endless-0', version: libendless_req)
 shard = dependency('eos-shard-0', version: shard_req)
 soup = dependency('libsoup-2.4')
 xapian_glib = dependency('xapian-glib-2.0')
+zim_glib = dependency('zim-glib-3.0')
```

Comment:
Extremely pedantic detail: these dependencies aren't required for the code I see in this PR (yet). But if you're planning to add the Zim stuff in the same branch, carry on! :)

Reply:
My bad! this should be moved to a different commit.